src/test/java/com/algorand/algosdk/cucumber/shared/TransactionSteps.java

```diff
@@ -82,6 +90,8 @@ public void buildApplicationTransactions(String operation, Long applicationId, S
         }
 
         builder.sender(sender);
+        builder.flatFee(fee);
+
```

Comment:
```suggestion
```

nit: extra line

Reply:
One concern here is that in cucumber, there is no "not set" value so we've always used 0 instead. So in theory this could change the intent of some tests.

I suppose if all the tests pass we weren't using that feature for this field